tensorflow/lite/micro/kernels/cmsis_nn/add.cc

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 
 #include "tensorflow/lite/kernels/internal/reference/add.h"
 
-#include "CMSIS/NN/Include/arm_nnfunctions.h"
+#include "Include/arm_nnfunctions.h"
```

Comment:
I added a comment in cmsis_nn.inc to change the -I so that the include path starts with `cmsis_nn`.

Reply:
See my comment above